README.md

```diff
@@ -185,6 +185,31 @@ Use the `registrycredentials` subcommand to to get registry credentials, passing
 
 * Show registry credentials for a specific account `ocm support get registrycredentials [accountID]`
 
+#### Getting access review
+
+Use the `accessreview` subcommand to check access permissions for a user on a resource, passing username, action, and resource type. Optional flags can provide additional context for scoped resources.
+
+The following flags are available for `get accessreview`:
+
+```
+--subscription-id    Subscription ID for subscription-scoped resources
+--organization-id    Organization ID for organization-scoped resources  
+--cluster-id         Cluster ID for cluster-scoped resources
+--suggest-roles      If true, provides general role suggestions when access is denied (refer to UHC account manager for accurate mappings)
+-h, --help           help for accessreview
+```
+
+##### Examples
+
+* Check if user can create ClusterTransfer: `ocm support get accessreview myuser CreateAction ClusterTransfer`
+* Check if user can get SubscriptionResource with subscription context: `ocm support get accessreview alice get SubscriptionResource --subscription-id sub-123`
+* Check if user can delete OrganizationResource with organization context: `ocm support get accessreview bob delete OrganizationResource --organization-id org-456`
+* Check if user can access cluster with multiple contexts: `ocm support get accessreview charlie get Cluster --subscription-id sub-123 --cluster-id cluster-789`
+* Get role suggestions when access is denied: `ocm support get accessreview myuser CreateAction ClusterTransfer --suggest-roles`
```

Comment:
I noticed that  ClusterTransfer doesn't have the Resource sufix. Is this flexible?